calculator.go

```diff
@@ -60,7 +60,7 @@
 func call(funcName string, args []float64) (float64, error) {
 	f, ok := functions[funcName]
 	if !ok {
-		return 0, fmt.Errorf("unknown function %s", funcName)
+		return 0, fmt.Errorf("unknown function %s", funcNam)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the variable name. It should be `funcName` instead of `funcNam`.
```diff
-		return 0, fmt.Errorf("unknown function %s", funcNam)
+		return 0, fmt.Errorf("unknown function %s", funcName)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->